src/semeio/fmudesign/_excel_to_dict.py

```diff
@@ -228,35 +228,39 @@ def _excel_to_dict_onebyone(
     except ValueError:
         output[key] = generalinput[key]  # Validation should raise
 
-    if "rms_seeds" in generalinput:
-        if isinstance(generalinput["rms_seeds"], str):
-            output["seeds"] = resolve_path(input_filename, generalinput["rms_seeds"])
-        else:
-            output["seeds"] = generalinput["rms_seeds"]
-    else:
-        output["seeds"] = None
+    key = "seed"
+    try:
+        output[key] = resolve_path(input_filename, str(generalinput["rms_seeds"]))
+    except KeyError:
+        output[key] = None
+    except (ValueError, TypeError):
+        output[key] = generalinput["rms_seeds"]  # Validatetion done later
 
-    # TODO: If not specified, should we seed with 0 instead of None?
-    # Not reproducible right now since we set default_rng(None)
     key = "distribution_seed"
     try:
         output[key] = int(generalinput[key])
-    except (KeyError, TypeError):
-        output[key] = None
-    except ValueError:
+    except KeyError as err:
+        raise ValueError(
+            f"Missing key: {key}, please specify a value.\nValid types: None or class int."
```

Comment:
I think we should improve this, e.g.:

```
You did not specify a value for 'distribution_seed', which is 
used to seen the random number generator that draws from
distributions in Monte Carlo sensitivities.
- Specify a number (e.g. a 6 digit integer) to seed the random number generator and obtain reproducible results.
- Specify 'None' if you do not want to seed the random number generator. Your analysis will not be reproducible.
```

This is now enforced whereas it was not before, so users need a good message